src/pages/fresh_install/FreshInstallPage.css

```diff
@@ -0,0 +1,100 @@
+.DivButtons {
+    margin-top: 3vh;
+}
+
+.ButtonCelsius {
+    margin-right: 2rem;
+}
+
+.Button {
+    width: 120px;
+    height: 45px;
+    object-fit: contain;
+    border-radius: 15px;
+    box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.15);
+    background-color: rgba(255, 255, 255, 0.9);
+    display: flex;
+    align-items: center;
+}
+
+.ButtonActive {
+    width: 120px;
+    height: 45px;
+    object-fit: contain;
+    border-radius: 15px;
+    box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.15);
+    background-color: rgba(255, 255, 255, 0.35);
+    display: flex;
+    align-items: center;
+}
+
+.DivContent {
+    text-align: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.Next {
+    display: flex;
+    align-items: center;
+    height: 54px;
+    color: #ffffff;
+}
+
+.DivNext {
+    padding-top: 26vh;
+}
+
+.Description {
+    display: flex;
+    align-items: center;
+    height: 54px;
+    color: #ffffff;
+}
+
+.DivDescription {
+    padding-top: 40vh;
+}
+
+.Text {
+    font-family: Avenir;
+    font-size: 20px;
+    font-weight: 500;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    text-align: center;
+}
+
+.ButtonText {
+    width: 120px;
+    height: 25px;
+    color: #2f72bb;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.TextActive {
```

Comment:
What CSS naming convention we are using on this project? I was not able to identify any convention...

As a suggestion, I'd recommend using [BEM](http://getbem.com/naming/) that is being very useful for other projects we are working on.